File: CS50x/Source Code/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //prompt user for input until valid data is entered.
    int  height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i <= height - 1; i++)
    {
        for (int j = 0; j <= (height + 2) + i; j++)
        {
            if (j < (height - 1) - i || (j >= height && j < height + 2))
            {
                printf(" "); //j++
            }
            else
            {
                printf("#"); //j++
            }
        }
        printf("\n"); //i++
    }
}

File: CS50x/Source Code/pset6/mario/less/mario.py
from cs50 import get_int

while True:
    h = get_int("height: ")
    if h > 0 and h < 9:
        break

for i in range(h):
    for j in range(h):
        if j < (h - 1) - i:
            print(" ", end="")
        else:
            print("#", end="")
    print()